config.toml

```diff
@@ -92,17 +106,22 @@ algolia_docsearch = false
 
 #Enable offline search with Lunr.js
 offlineSearch = true
+offlineSearchSummaryLength = 70
+offlineSearchMaxResults = 10
 
-github_branch= "main"
+# Enable syntax highlighting and copy buttons on code blocks with Prism
+prism_syntax_highlighting = true
 
 # User interface configuration
 [params.ui]
 # Enable to show the side bar menu in its compact state.
 sidebar_menu_compact = true
+sidebar_menu_foldable = false
```

Comment:
I've been wondering about this one, maybe it would be useful for use as like under `/users` we have lots of entries, it may make things a bit cleaner/more visible

Reply:
There is a massive change coming on how sections are organized and how pages are written (also to make the left-side navigation tree happy and work optimally).